README.md

```diff
@@ -43,12 +43,21 @@ Dockerized CLI application which allows to automatically sync different resource
    ```bash
    usage: commercetools-project-sync
     -h,--help         Print help information.
-    -s,--sync <arg>   Choose which sync module to run: "types", "productTypes", "categories", "products" or
-                      "inventoryEntries".
+    -s,--sync <arg>   Choose which sync module to run: "types", "productTypes", "categories", "products", "inventoryEntries" or "all".
```

Comment:
As I understand currently there is no support to run for instance `types` and `productTypes` only. So maybe it makes sense to rephrase the sentence?

```suggestion
    -s,--sync <arg>   Choose one of the following modules to run: "types", "productTypes", "categories", "products", "inventoryEntries" or "all" (will run all the above).
```

wdyt?

Reply:
I like the suggestion! thanks!

Reply:
c75a859